examples/listqa/list_nq2tydi.py

```diff
@@ -0,0 +1,261 @@
+# this code will get the NQ answers that contain lists and convert the lists from NQ format to TyDi format. 
+# Note: This is a heuristic and there may be some non-list questions. 
+# A question is considered to have a list as an answer if there is a list in the paragraph and no short answer.
+# The paragraph offsets are used as the "short answer".
+
+import json
+import gzip
+from copy import deepcopy
+from dataclasses import dataclass, field
+import logging
+from tabnanny import verbose
+from transformers import HfArgumentParser
+import sys, os
+from tqdm import tqdm
+import glob
+
+@dataclass
+class List2TyDiArguments:
+    """
+    Arguments pertaining to processing nq.
+    """
+    input_file: str = field(
+        default='/NQ/download/dir/here', metadata={"help": "input file(s) of nq data in original format."}
+    )
+    output_file: str= field(default='/output/loc/here/jsonl', metadata={"help": "directory to output file(s) in tydi google format. (jsonl)"}
+    )
+    num_lines: int = field(default=-1, metadata={"help":"number of examples in the dataset input file (7830 for NQ dev), -1 if unknown or multiple files at once."}
+    )
+    verbose: bool = field(
+        default=False, metadata={"help": "Print full examples"}
+    )
+
+class ListNQSubset:
+    """
+    Class to process the full NQ dataset and get the list subset
+
+    main function is process which requires the following inputs:
+
+    input_file = location of NQ dataset file(s)
+    output_file = location for output
```

Comment:
Now that @rongzhangibm has gotten NQ to work do you want to use the HF dataset similarly to how he's using?

Reply:
> Overall LGTM to me. I added a minor comment which you can address via an issue that you create and work on. Let's wait for the CI to finish and we'll be good to approve after 2 more approves from others.

Sounds good. I will work on this as a new issue. This will be v2.